src/main/python/pre_fx_data.py

```diff
@@ -286,8 +283,14 @@ def run_extraction_and_auto_qc(self, nwb_path, stimulus_ontology, qc_criteria, c
 
             self.sweep_features = sweep_features
             self.sweep_states = sweep_states
+            self.manual_qc_states = {sweep["sweep_number"]: "default" for sweep in self.sweep_features}
+
+            self.end_commit_calculated.emit(
+                self.sweep_features, self.sweep_states, self.manual_qc_states, self.data_set
+            )
 
-            self.end_commit_calculated.emit()
+    def on_manual_qc_state_updated(self, sweep_number: int, new_state: str):
+        self.manual_qc_states[sweep_number] = new_state
 
 
 def extract_qc_features(data_set):
```

Comment:
To make it work I had to add
`sp.drop_tagged_sweeps(sweep_features)`
after
`sweep_features = sweep_qc_features(data_set)`